nbformat/tests/test_sign.py

```diff
@@ -4,9 +4,11 @@
 # Distributed under the terms of the Modified BSD License.
 
 import copy
+import os
 import shutil
 import time
 import tempfile
+import testpath
```

Comment:
Sorry for the confusion - that link to the notebook setup.py was meant as a *reference* for how to add testing requirements.

You will have to add a line after the existing `pip install` with:

    pip install nbformat[test]

in `.travis.yml` for it to have the desired effect.